package.json

```diff
@@ -13,7 +13,11 @@
     "build:prepare": "esno scripts/prepare.ts",
     "build:web": "vite build",
     "build:js": "vite build --config vite.config.content.ts",
-    "pack": "crx pack extension -o ./extension.crx",
+    "pack": "cross-env NODE_ENV=production run-s run-p pack:*",
+    "pack:crx": "crx pack extension -o ./extension.crx",
```

Comment:
It seems that `web-ext` supports `chromium-based` browsers, can we use it to replace `crx pack`? Have you tested and verified this situation?

https://extensionworkshop.com/documentation/develop/web-ext-command-reference/#:~:text=Here%20are%20the%20supported%20targets

Reply:
`web-ext build` currently only supports output `.xpi` and `.zip` fromat.